src/app/styles/variables/colors.scss

```diff
@@ -0,0 +1,27 @@
+$color_background-body: #fff;
+$color_background-button: #0a396d;
+
+$color_text-main: #001936;
+$color_text-title: #333;
+$color_text-contrast: #8b8d91;
+$color_text-input: #666;
+$color_text-input-focus: #111;
+
+$color_contrast-hover: #0e649b;
+$color_accent: #0a396d;
+
+$color_link: $color_accent;
+$color_link-visited: $color_text-main;
+$color_link-hover: $color_contrast-hover;
+
+$color_border-button: #ccc;
+$color_border-button-hover: #aaa;
+$color_border-button-focus: #bbb;
+$color_border1: #e1e1e1;
+$color_border2: #666;
+
+
+// $color__border-button: #ccc #ccc #bbb;
```

Comment:
remove or uncomment; we do not commit commented code

Reply:
Unnecessary commented code removed.